packages/expo-cli/src/credentials/provider/ios.ts

```diff
@@ -0,0 +1,135 @@
+import { SetupIosProvisioningProfile } from '../views/SetupIosProvisioningProfile';
+import { SetupIosDist } from '../views/SetupIosDist';
+import { runCredentialsManager } from '../route';
+import { Context } from '../context';
+import { credentialsJson } from '../local';
+import { CredentialsProvider } from './provider';
+
+export interface iOSCredentials {
+  provisioningProfile: string;
+  distributionCertificate: {
+    certP12: string;
+    certPassword: string;
+  };
+}
+
+interface Options {
+  projectName: string;
+  accountName: string;
+  bundleIdentifier: string;
+}
+
+export class iOSCredentialsProvider implements CredentialsProvider {
+  public readonly platform = 'ios';
+  private readonly ctx = new Context();
+  private credentials?: iOSCredentials;
+
+  constructor(private projectDir: string, private options: Options) {}
+
+  get projectFullName(): string {
+    const { projectName, accountName } = this.options;
+    return `@${accountName}/${projectName}`;
+  }
```

Comment:
Maybe we already have a function like this in `@expo/config`. If we don't - maybe we should move it there. This is not iOS-specific.

Reply:
Explained in AndroidCredentialsProvider